.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
```

Comment:
[This code](https://github.com/Parsely/wp-parsely/blob/develop/.github/workflows/integration-tests.yml#L7-L8) aims to _not_ run the PHP tests when the commit only contains Markdown files.

Would it be possible to limit this Node.js Test workflow to only run if a (non-built) `.js` was updated? Could a change in a PHP file make JS tests fail?

Reply:
By only testing `push` when it's the `develop` branch, then Node.js tests won't be run on feature branch pushes; they will only run when the PR is opened (as we don't `push` directly to `develop`. Is that the intent? Or should `push` be opened for all branches?